package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "concurrently",
-      "version": "7.2.0",
+      "version": "7.2.1",
```

Comment:
undo?

Reply:
I'd usually do it via `npm version` so that it bumps the version, creates a commit and tags it all at once